src/app/dashboard/dashboard.component.html

```diff
@@ -22,12 +23,15 @@ <h2>{{ projectBuild.project.name }}</h2>
     </div>
   </section>
 
-  <section class="columns" *ngIf="projectsBuilds.length == 0">
+  <section class="columns" *ngIf="projectsBuilds?.length == 0">
```

Comment:
The only real change here is the `?` null check... yet, vscode seems to have made some auto-formatting changes, perhaps on account of the `.vscode` settings in this repo?